python/lsst/ip/diffim/zogy.py

```diff
@@ -288,8 +319,8 @@ def computePrereqs(self, psf1=None, psf2=None, padSize=0):
         padSize = self.padSize if padSize is None else padSize
         Pr, Pn = psf1, psf2
         if padSize > 0:
-            Pr = ZogyTask._padPsfToSize(psf1, (padSize, padSize))
-            Pn = ZogyTask._padPsfToSize(psf2, (padSize, padSize))
+            Pr = ZogyTask._padPsfToSize(psf1, (psf1.shape[0]+padSize, psf1.shape[0]+padSize))
```

Comment:
Did you mean: `(psf1.shape[0]+padSize, psf1.shape[1]+padSize)` or is there something special about the `x` dimension. Please comment if so.